packages/angular-output-target/resources/control-value-accessors/boolean-value-accessor.ts

```diff
@@ -15,7 +15,7 @@ import { ValueAccessor } from './value-accessor';
       useExisting: BooleanValueAccessor,
       multi: true
     }
-  ]
+  ]<VALUE_ACCESSOR_STANDALONE>
```

Comment:
For readability, it might be clearer to update this so that we only replace the boolean value for the `standalone` option. So something like:

```suggestion
  ],
  standalone: <VALUE_ACCESSOR_STANDALONE>
```

And then update the replacement logic when generating the accessor.

Reply:
I wouldn't apply this suggestion. I've seen there's a dependency on Angular 8. This standalone feature wasn't implemented in Angular 8 hence it could throw errors. So for backwards compatibility I would keep it as is. What do you think?

Reply:
Ah, good call! Makes sense!